charts/fpl-case-service/values.preview.template.yaml

```diff
@@ -61,7 +61,7 @@ xui-webapp:
       SERVICES_CCD_DATA_STORE_API: http://${SERVICE_NAME}-ccd-data-store-api
       SERVICES_TERMS_AND_CONDITIONS: http://xui-terms-and-conditions-${SERVICE_FQDN}.service.core-compute-preview.internal
       JURISDICTIONS: PUBLICLAW
-      LAUNCH_DARKLY_CLIENT_ID: dummy
+      LAUNCH_DARKLY_CLIENT_ID: 5de6610b23ce5408280f2268
```

Comment:
Do we want this to be public?

Reply:
ideally not, this is xui id, exposed by xui publicly 
https://github.com/hmcts/rpx-xui-

webapp/blob/27b2eb59e045a1f72796b079712fe099089c0c19/charts/xui-webapp/values.preview.template.yaml#L9